src/garmin/GarminConnect.js

```diff
@@ -243,6 +243,11 @@ class GarminConnect {
         return this.client.get(url, data);
     }
 
+    async downloadOriginalActivityData(dir, activityId) {
```

Comment:
Maybe use { activityId } as an argument, to be consistent with other methods where a whole activity can be supplied.